packages/react-utils/src/isRefObject.test.ts

```diff
@@ -0,0 +1,35 @@
+import { createRef } from 'react'
+import { describe, expect, it } from 'vitest'
+import { isRefObject } from './isRefObject'
+
+describe('isRefObject', () => {
+  it('returns true for a createRef object', () => {
+    const ref = createRef<HTMLDivElement>()
+    expect(isRefObject(ref)).toBe(true)
+  })
```

Comment:
Interestingly the first test is failing. `Object.isExtensible(value)` inside `isRefObject` fails the test. Care to look at it @Dashice ? :)